### ISSUE TITLE
UI/UX difficulties when working with different credential formats that mean the same credential type

### ISSUE THREAD

**Issue by JoTiTu** [openid/OpenID4VCI#400]
From an UI perspective a wallet displays Credential Types and the user does not have to know if it is a mDoc or Sd-JWT credential. So the user just cares if it is for example a CreditCard. This is easy as long as the credential is used either as a Sd-Jwt OR mDoc.

Assuming an issuer issues the Credentials Type CreditCard in two different formats at the same time (mDoc AND Sd-Jwt) using a different schema for each of them, this gets more difficult because now the wallet has to know that both schemas semantically refer to the same Credential Type. This link between two schemas that refer to the same type is important so the wallet can make sure to map them and only show the Credential Type in the wallet once.

As of my knowledge currently there is no such feature but it would make the life of wallet implementers a lot easier since as of now workarounds are necessary to create groups.

So basically it would need some kind of link between two credential_supported_id.

**Comment by jogu** [comment]
I can see the potential problem here.

It may not be ideal, but my initial reaction would be that `display.name` in the two entries in `credential_configurations_supported` would be the same - could the wallet use that as a signal?

**Comment by JoTiTu** [comment]
That might be true in some cases but I think that this wouldn't be a bulletproof solution and more of a workaround.
A issuer could issue two different credentials that might share some similar fields even though they are semantically different.
I think a dedicated solution to address this grouping problem would be beneficial.

**Comment by Sakurann** [comment]
how the wallet manages that two credential types of different formats are the same "thing" is up to the wallet so I also do not see an interoperability problem. sure, somehow the wallet need to learn which credential types of different format are the same "thing". and right now that is being done out of band and since I don't think we expect a lot of credential types being issued in multiple formats (So far PID in EUDIW is the main one I know?), that should not be a big issue either. in short, I am not convinced this is the problem we need to solve in the specification.
